packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelController.java

```diff
@@ -177,6 +182,12 @@ ResponseEntity<Model> getModel(
 			if (model.isEmpty()) {
 				return ResponseEntity.noContent().build();
 			}
+			// GETs not associated to a projectId cannot read non public or temporary assets
+			if (permission.equals(Schema.Permission.NONE)
+					&& !model.get().getPublicAsset()
+					&& !model.get().getTemporary()) {
```

Comment:
Does this imply that any temp asset is essentially public?

Reply:
Nope, temporary and public are two different things.
A public asset is one that do not belong to a project and is not editable.
A temporary assets are the ones created by workflow operators and not saved by the user within the project.

The condition above reads as:
> if their is no permission on the private and permament asset.

Reply:
No - I think I am implying that people without permissions can only see assets that are both Public and not temporary - do I have the logic incorrect here?  (reading again)